apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -1695,6 +1695,8 @@ public static void workAroundHAPIIssue1585(RequestDetails requestDetails) {
   }
 
   /**
+   * TODO: Remove this method when the calling method has been removed as per BFD-XXXX
```

Comment:
Do we have a story made for this? (It seems like we should so we can note it here)

Reply:
(I said no blockers but this is semi-blocker as we should avoid putting in TODOs without an actionable story attached IMO)

Reply:
thanks! I meant to go find that and replace the XXXX and completely forgot (we do have a story for it somewhere).